app/controllers/home_controller.rb

```diff
@@ -18,9 +18,11 @@ def sell; end
 
   private
 
-  # Cache the value of the query count for 24 hours
+  # Cache the value of the query count
   def cached_count(key, query)
-    Rails.cache.fetch("home_stats_count_#{key}", expires_in: 1.day, race_condition_ttl: 10) do
+    CacheService.cache("home_stats_count_#{key}",
+                       expires_in: CacheService::HOME_STATS_EXPIRY,
```

Comment:
if `CacheService` is the owner of that constant, we might as well not leak the `expires_in` outside, don't you think? I talking about having something like

```
CacheService.cache("home_stats_count_#{key}", race_condition_ttl: 10) do
end
```

while still having the option to override the `expires_in` if we need to.

Reply:
I was thinking about extracting the whole thing into a specific `#home_stats` method in `CacheService`...?